charm.go

```diff
@@ -38,3 +40,29 @@ func ReadCharm(path string) (charm Charm, err error) {
 	}
 	return charm, nil
 }
+
+// SeriesToUse takes a specified series and a list of series supported by a
+// charm and returns the series which is relevant.
+// If the specified series is empty, the any default series defined by the
+// charm is used. Any specified series is validated against those supported
```

Comment:
Reworded:

// If the requested series is empty, then any default series defined by the
// charm is used, otherwise the requested series is validated against those
// supported by the charm.